.github/workflows/install-mdbook/action.yml

```diff
@@ -27,3 +27,12 @@ runs:
     - name: Install i18n-helpers
       run: cargo install mdbook-i18n-helpers --locked --version '${{ steps.mdbook-i18n-helpers-version.outputs.version }}'
       shell: bash
+
+    - name: Install mdbook-pandoc and related dependencies
+      run: |
+        cargo install mdbook-pandoc --locked --version 0.5.0
```

Comment:
can we use https://github.com/marketplace/actions/install-cargo-binstall so that we don't need to compile from scratch?

Reply:
I'll add that later!

Reply:
My idea is to merge it as a foundation so we are getting started with it. Then I will open a polishing PR, because I saw while looking through the pdf, that there are some issues with line width, when it comes to code blocks.

Reply:
> there are some issues with line width, when it comes to code blocks

line wrapping in code blocks is coming [very soon](https://github.com/max-heller/mdbook-pandoc/pull/72/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR17)!

Reply:
I fixed it now by hand, but maybe I can just revert this, when it lands. Thanks for the update!

Reply:
Ah wait, I thought you linked to an issue :D if it's already there, it may not make sense at all to push the changes, and I may rather wait and see the outcome.